File: sdkwork-app-sdk-csharp/Models/CurrencyConvertVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class CurrencyConvertVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public string? BaseCurrencyCode { get; set; }
        public string? TargetCurrencyCode { get; set; }
        public double? OriginalAmount { get; set; }
        public double? Rate { get; set; }
        public double? ConvertedAmount { get; set; }
    }
}

File: sdkwork-app-sdk-typescript/src/types/currency-convert-vo.ts
/** 货币兑换响应 */
export interface CurrencyConvertVO {
  /** 创建时间 */
  createdAt?: string;
  /** 更新时间 */
  updatedAt?: string;
  /** 基础货币代码 */
  baseCurrencyCode?: string;
  /** 目标货币代码 */
  targetCurrencyCode?: string;
  /** 原始金额 */
  originalAmount?: number;
  /** 汇率 */
  rate?: number;
  /** 兑换后金额 */
  convertedAmount?: number;
}
